porting/common-kernel-build-errors.rst

```diff
@@ -15,4 +15,42 @@ If you receive something similar to the following error::
 Apply the patch `nick kvfree() from apparmor`_.
 
 
-.. _nick kvfree() from apparmor: https://github.com/ubports/android_kernel_moto_shamu/commit/83f949a8de673fe45499d1741da8654831a5afae
\ No newline at end of file
+.. _nick kvfree() from apparmor: https://github.com/ubports/android_kernel_moto_shamu/commit/83f949a8de673fe45499d1741da8654831a5afae
+
+
+'kuid_t' (sdcardfs, cgroup) error 
+----------------------------------
+
+Example of the error::
+
+   ../../../../../../kernel/lenovo/msm8916/kernel/cgroup.c:2138:37: error: invalid operands to binary != (have 'kuid_t' and 'kuid_t')
+   if (current != task && cred->euid != tcred->uid &&
+
+   ../../../../../../kernel/lenovo/msm8916/kernel/cgroup.c:2139:18: error: invalid operands to binary != (have 'kuid_t' and 'kuid_t')
+       cred->euid != tcred->suid)
+       
+Set 'CONFIG_USER_NS' to 'n' in your defconfig.
```

Comment:
double backticks for inline code

Set ``CONFIG_USER_NS`` to ``n`` in your defconfig.